src/js/actions/tool/pen.js

```diff
@@ -90,7 +90,11 @@ define(function (require, exports) {
         if (vectorMode) {
             selectVectorMask = descriptor.playObject(vectorMaskLib.activateVectorMaskEditing());
         } else {
-            selectVectorMask = descriptor.playObject(vectorMaskLib.dropPathSelection());
+            var applicationStore = this.flux.store("application"),
+                currentDocument = applicationStore.getCurrentDocument();
+            if (currentDocument) {
+                selectVectorMask = descriptor.playObject(vectorMaskLib.dropPathSelection());
```

Comment:
It may well be perfectly fine to not initialize this variable if there is no current document w.r.t. `Promise.join`, but... we are usually careful to do this elsewhere. It seems like we should be consistent.